src/modules/config-panel/config-panel.ts

```diff
@@ -10,6 +11,11 @@ import {NotificationService} from '../notification/notification.service';
 
 @inject(Router, 'NotificationService', EventAggregator, 'AuthenticationService', OpenIdConnect, 'InternalProcessEngineBaseRoute')
 export class ConfigPanel {
+  public isLoggedInToProcessEngine: boolean;
+  @bindable() public baseRoute: string;
```

Comment:
Just somehting minor, you can use the `()` after `@bindable` but you don't have two if you are not using the options of bindable.

```suggestion
  @bindable public baseRoute: string;
```